### PR TITLE
Replace jenia tags

### PR DESCRIPTION
**Pull request by solth**
- replace Jenia `jp:popupFrame` components with Primefaces `p:dialog`s in `BenutzerBearbeiten.xhtml`, `ProjekteBearbeiten.xhtml`, `schritt_box_Benutzer.xhtml` and `schritt_box_Benutzergruppen.xhtml`
- replace Tomahawk `t:inputCalendar` with Primefaces `p:calendar` components because Tomahawk popup calenders cause JS conflicts with jQuery, which is used by Primefaces components

### CODE REVIEW

File: Kitodo/src/main/webapp/newpages/ProjekteBearbeiten.xhtml [hunk 1]
```diff
@@ -1072,12 +163,12 @@
                 <ui:include src="inc/tbl_Fuss.xhtml" />
             </table>
 
-            <script type="text/javascript">
-                //<![CDATA[
-                    var tabView1 = new YAHOO.widget.TabView('projectform:demo');
-                    var tabView2 = new YAHOO.widget.TabView('projectform:staticticdemo');
-                //]]>
-            </script>
+            <!--<script type="text/javascript">-->
```
Comment (henning-gerhardt) [inline Kitodo/src/main/webapp/newpages/ProjekteBearbeiten.xhtml]: If this JavaScript is not needed any more, remove it.

File: Kitodo/src/main/webapp/newpages/inc_editProjects/detailsTab.xhtml [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+            <h:panelGroup id="id18">
+                <h:inputText id="titel"
+                             style="width: 200px;margin-right:15px"
+                             value="#{ProjekteForm.myProjekt.title}"
+                             required="true"/>
+                <t:message id="id19" for="titel" style="color: red"
+                           replaceIdWithLabel="true"/>
+            </h:panelGroup>
+
+            <!-- Mets als internes Speicherformat (ansonsten xStream)
```
Comment (henning-gerhardt) [inline Kitodo/src/main/webapp/newpages/inc_editProjects/detailsTab.xhtml]: Why are you adding out commented lines to a new file? I think this should be removed or an explanation should be added.

File: Kitodo/src/main/webapp/newpages/inc_editProjects/detailsTab.xhtml [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+                <td>
+                    <h:outputText id="vwid35"
+                                  value="#{ProjekteForm.calcPagesPerDay}"/>
+                </td>
+            </tr>
+
+            </tbody>
+        </table>
+    </div>
+</ui:composition>
```
Comment (henning-gerhardt) [inline Kitodo/src/main/webapp/newpages/inc_editProjects/detailsTab.xhtml]: Missing new line at end of file.

File: Kitodo/src/main/webapp/newpages/inc_editProjects/metsParameterTab.xhtml [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+
+            <!-- metsPointerPathAnchor -->
+            <h:outputText id="id73"
+                          value="#{msgs.metsPointerPathAnchor}"/>
+            <h:inputText id="id74"
+                         style="width: 650px;margin-right:15px"
+                         value="#{ProjekteForm.myProjekt.metsPointerPathAnchor}"/>
+
+            <!-- metsPurl -->
+
```
Comment (henning-gerhardt) [inline Kitodo/src/main/webapp/newpages/inc_editProjects/metsParameterTab.xhtml]: Remove this empty line to look same as other parts.

File: Kitodo/src/main/webapp/newpages/inc_editProjects/metsParameterTab.xhtml [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+                              modal="true" dynamic="true"
+                              onHide="#{NavigationForm.Reload()}">
+                        <ui:include
+                                src="/newpages/ProjektFilegroupPopup.xhtml"/>
+                    </p:dialog>
+                </h:panelGroup>
+            </h:panelGroup>
+        </h:panelGrid>
+    </div>
+</ui:composition>
```
Comment (henning-gerhardt) [inline Kitodo/src/main/webapp/newpages/inc_editProjects/metsParameterTab.xhtml]: New line is missing at end of file.

File: Kitodo/src/main/webapp/newpages/inc_editProjects/statisticsTab.xhtml [hunk 6]
```diff
@@ -0,0 +1,292 @@
...
+                    <!-- throughputPerDay -->
+                    <h:outputText id="qqvwid32"
+                                  value="#{msgs.pagesPerDay}:"/>
+                    <h:outputText id="qqvwid33"
+                                  value="#{ProjekteForm.calcPagesPerDay}"/>
+                </h:panelGrid>
+            </h:panelGrid>
+        </h:panelGroup>
+
+        <!--<htm:div id="staticticdemo" styleClass="yui-navset">-->
```
Comment (henning-gerhardt) [inline Kitodo/src/main/webapp/newpages/inc_editProjects/statisticsTab.xhtml]: Is this line needed? If not remove it.

File: Kitodo/src/main/webapp/newpages/inc_editProjects/statisticsTab.xhtml [hunk 7]
```diff
@@ -0,0 +1,292 @@
...
+                                    src="/newpages/inc_statistic/StatisticStorage.xhtml"/>
+                        </f:subview>
+                    </t:aliasBeansScope>
+                </div>
+            </t:panelTab>
+
+        </t:panelTabbedPane>
+
+    </div>
+</ui:composition>
```
Comment (henning-gerhardt) [inline Kitodo/src/main/webapp/newpages/inc_editProjects/statisticsTab.xhtml]: New line at end of file is missing.

File: Kitodo/src/main/webapp/newpages/inc_editProjects/technicalDataTab.xhtml [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+                <h:inputText id="timeout"
+                             style="width: 300px;margin-right:15px"
+                             value="#{ProjekteForm.myProjekt.dmsImportTimeOut}"
+                             required="true"/>
+                <t:message id="id52" for="timeout" style="color: red"
+                           replaceIdWithLabel="true"/>
+            </h:panelGroup>
+        </h:panelGrid>
+    </div>
+</ui:composition>
```
Comment (henning-gerhardt) [inline Kitodo/src/main/webapp/newpages/inc_editProjects/technicalDataTab.xhtml]: New line at end of file is missing.

File: Kitodo/src/main/webapp/newpages/inc_editProjects/statisticsTab.xhtml (hunk 6 above)
Comment (solth) [inline Kitodo/src/main/webapp/newpages/inc_editProjects/statisticsTab.xhtml]: This will be removed in a separate PR to clean up Yahoo javascript residue.
